ui/components/institutional/institutional-components.scss

```diff
@@ -0,0 +1,11 @@
+/**
+* Please import your styles in order of atomicity.
+* The most atomic styles should be imported first.
+* This will help improve specificity and reduce the chance of
+* unintended overrides.
+**/
+@import 'compliance-details/index';
+@import 'compliance-settings/index';
+@import 'jwt-dropdown/jwt-dropdown';
+@import 'jwt-url-form/jwt-url-form';
+@import 'note-to-trader/index';
```

Comment:
Not all of these files are introduced here, i'm assuming that the ones that are being imported here were added in previous PRs?

Reply:
Yes, that's correct